packages/tokens/style-dictionary.config.js

```diff
@@ -51,6 +51,18 @@ const EDSStyleDictionary = StyleDictionary.extend({
             },
           },
         },
+        {
+          format: "javascript/es6",
+          destination: "ts/colors.ts",
```

Comment:
Can we avoid needing the duplicate TS file by setting `checkJs: true` in the tsconfig?

Reply:
I tried turning on `allowJs` & `checkJs` but still saw the error 😬 

<img width="517" alt="Screen Shot 2021-07-29 at 12 30 59 PM" src="https://user-images.githubusercontent.com/15840841/127530077-4cefccbd-8d04-4504-8f33-60ab87b13682.png">